src/components/wallet/wallet.ts

```diff
@@ -40,6 +40,7 @@ interface StellarAccount {
   publicKey: string
   secretKey: string
   state?: ServerApi.AccountRecord
+  claimableBalances?: ServerApi.ClaimableBalanceRecord
```

Comment:
What is `ServerApi`?, it's not great that things are usable in a file without being imported.

Reply:
I was more or less following the logic that Tyler set up for this interface.
The `ServerApi` is imported on line 2 of the file.

Reply:
Oh I missed that, whoops